web_app/telegram/handlers/menu.py

```diff
@@ -0,0 +1,125 @@
+from aiogram import Router, types, Bot, F
+from sqlalchemy import select, delete
+from sqlalchemy.orm import Session
+
+from database.models import NotificationData
+from .utils import kb
+
+menu_router = Router()
+
+
+@menu_router.callback_query(F.data == "go_menu")
+async def menu(callback: types.CallbackQuery, db: Session, bot: Bot):
+    """
+    This function is called when the user clicks the "go_menu" button in Telegram.
+    It sends the user a message "Menu:" and displays a menu with buttons.
+    """
+    await callback.message.edit_text("Menu:", reply_markup=kb.menu())
+
+
+@menu_router.callback_query(F.data.startswith("notification_delete_confirm_"))
+async def delete_notification_confirm(callback: types.CallbackQuery, db: Session):
+    """
+    This function is called when the user confirms the deletion of a notification.
+    It deletes the notification from the database and sends the user a message about successful deletion.
+    """
+    # get the notification identifier
+    ident = callback.data.removeprefix("notification_delete_confirm_")
+    # delete the notification
+    stmp = delete(NotificationData).where(NotificationData.id == ident)
```

Comment:
The same thing here, I would love to see it in `crud` all operation with DB nice to have in one place.
It's better for maintance